src/main/java/com/example/school/model/Group.java

```diff
@@ -0,0 +1,19 @@
+package com.example.school.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+public class Group {
+    private UUID id;
+    private String name;
+    private List<Student> students;
```

Comment:
It would be better to use here `@Builder.Default` from lombok(read additional info in internet) and change this line in this way:
`private List<Student> students = new ArrayList<Student>();`.
And after you can stop using `.students(new ArrayList<>())` when you create `Group` throw builder in `SchoolApplication` class.